db/payment_icons.yml

```diff
@@ -2282,3 +2282,7 @@
   name: payrexxbanktransfer
   label: Payrexx Bank Transfer
   group: bank_transfers
+-
+  name: zustaina
+  label: Zustaina
+  group: other
```

Comment:
Is Zustaina a gift card offering? If so, please use this group: gift_card

Reply:
> Is Zustaina a gift card offering? If so, please use this group: gift_card

I've changed the group but now I get the error during the checks:

```
PaymentIconTest#test_Every_payment_icon_record_is_valid [test/unit/payment_icon_test.rb:14]:
Expected [:credit_cards, :cryptocurrencies, :bank_transfers, :wallets, :convenience_stores, :other] to include :gift_card.
```